lib/puppet/application/filebucket.rb

```diff
@@ -69,6 +69,14 @@ def help
 use your local file bucket by specifying '--local', or by specifying
 '--bucket' with a local path.
 
+> **Note**: Enabling and using the backup option, and by extension the 
+  filebucket resource, requires appropriate planning and management to ensure 
+  that sufficient disk space is available for the file backups. Generally, you 
+  can implenent this using one of the following two options:
```

Comment:
nit `implement`, same comment in `file.rb`

Reply:
Good catch! I'll push up an edit